app/js/lib/foxbox/foxbox.js

```diff
@@ -123,49 +127,66 @@ export default class Foxbox extends Service {
    * @returns {Promise}
    * @private
    */
-  _initDiscovery() {
+  _initDiscovery(timeout) {
```

Comment:
I agree with @azasypkin but that'd be nice to land in this patch so that we don't pollute too much git history and blame.